consensus/istanbul/backend/announce.md

```diff
@@ -0,0 +1,140 @@
+# Celo Announce Protocol
+
+While here referred as a protocol by itself, Announce is technically a subset of the Istanbul protocol, on the [RLPx] transport, that facilitates Validators to privately discover the eNodeURL of other Validators in the network, using the underlying p2p eth network. This is used by Istanbul to have direct connections between Validators, speeding up the consensus phase of block generation.
+
+It is important to note that a validator's `eNodeURL` is never shared among third parties, but directly published to specific peers. Therefore it is allowed for a validator to publish different external facing `eNodeURL` values to different validator peers.
```

Comment:
I this needs explanation to be expanded a bit. If I understand correctly a validator will share their enode via the normal discovery mechanism. But the key used in that enode is ephemeral (or at least more ephemeral than the validator key), so although the validator's enode is shared among third parties, those third parties can't tell if the enode belongs to a validator or not.

Reply:
I think a section just outlining how the two different keys come into play could be useful

Reply:
i guess you could say that:

* any node has a enodeURL, that enodeURL is public and is shared through discovery.
* a validator node, has also an pk/address pair that identifies itself
* announce is about revealing the mapping `valdiatorAddress => enodeURL` so nodes can easily identify/connect which nodes are validators.

I would speak to lengthy about this. enodeURL are already explained on the devp2p protocol, just a refresher. the concept that a node has an etherbase and enodeURL is confusing for most, but because they haven't read the specs ;)

Reply:
@hbandura @mcortesi As this stands I think this section is misleading.

The following phrase to me seems incorrect
>It is important to note that a validator's `eNodeURL` is never shared among third parties

Validator's `eNodeURLs` are shared with everyone.